readthedocs/notifications/storages.py

```diff
@@ -29,17 +38,26 @@ class FallbackUniqueStorage(FallbackStorage):
     render the text in templates.
     """
 
+    storages_names = (
+        'readthedocs.notifications.storages.NonPersistentNotification',
+        'messages_extends.storages.StickyStorage',
+        'messages_extends.storages.PersistentStorage',
+        'django.contrib.messages.storage.cookie.CookieStorage',
+        'django.contrib.messages.storage.session.SessionStorage',
+    )
+
     def _get(self, *args, **kwargs):
         # The database backend for persistent messages doesn't support setting
         # messages with ``mark_safe``, therefore, we need to do it broadly here.
         messages, all_ret = (super(FallbackUniqueStorage, self)
                              ._get(self, *args, **kwargs))
+
         safe_messages = []
         for message in messages:
             # Handle all message types, if the message is persistent, take
             # special action. As the default message handler, this will also
             # process ephemeral messages
```

Comment:
Oh huh, it *was* written to handle all message types. Perhaps the library changed?